src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/BlobContainerClientBuilderProvider.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.blobrouter.services.storage;
+
+import com.azure.storage.blob.BlobContainerClientBuilder;
+import org.springframework.beans.factory.annotation.Lookup;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BlobContainerClientBuilderProvider {
+
+    @Lookup("bulk-scan-blob-client-builder")
+    public BlobContainerClientBuilder getBlobContainerClientBuilderBean() {
+        return null;
+    }
```

Comment:
This is probably fine. Now that we have this provider, I wonder if the builder could just be created here. Thoughts?

Reply:
changed